workflow-fingerprint-collector/src/main/java/ca/on/oicr/pde/workflows/FingerprintCollectorWorkflow.java

```diff
@@ -70,16 +59,17 @@
     private String bundledJRE;
     private String[] rgDetails;
     private boolean doBamFix = false;
+    private static final String INPUT_FILES="input_files";
 
     @Override
     public Map<String, SqwFile> setupFiles() {
         // Set up reference, bam and vcf files here
         try {
-            if (getProperty("input_files") == null) {
-                Logger.getLogger(FingerprintCollectorWorkflow.class.getName()).log(Level.SEVERE, "input_files is not set, we need at least one bam file");
+            if (getProperty(INPUT_FILES) == null) {
+                Logger.getLogger(FingerprintCollectorWorkflow.class.getName()).log(Level.SEVERE, INPUT_FILES+" is not set, we need at least one bam file");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)